src/common/utils/elevation.h

```diff
@@ -294,8 +294,12 @@ inline bool run_non_elevated(const std::wstring& file, const std::wstring& param
     siex.StartupInfo.cb = sizeof(siex);
 
     PROCESS_INFORMATION pi = { 0 };
+
+    auto cmd_line = std::make_unique<WCHAR[]>(executable_args.size()+1);
+    wcscpy_s(cmd_line.get(), executable_args.size() + 1, executable_args.c_str());
+
     auto succeeded = CreateProcessW(file.c_str(),
-                                    const_cast<LPWSTR>(executable_args.c_str()),
+                                    &cmd_line[0],
```

Comment:
These changes look more error prone just to remove the cast. Why is this needed?

Reply:
The second parameter for CreateProcessW is a LPWSTR [in, out, optional] parameter and may be modified by the function. wstring::c_str() returns a const wchar_t.
The const cast is being used to cast away const, lying to the compiler and is undefined behavior.
To provide CreateProcessW with something non-const a copy of the executable_args.c_str().

Reply:
As we don´t need a copy of the string,
I passing &executable_args[0], should do it.